cmd/build.go

```diff
@@ -123,6 +134,13 @@ func newBuildCmd() *cobra.Command {
 		launch.ArtifactsDir,
 		"Path to the host side directory which is mounted into $SD_ARTIFACTS_DIR.")
 
+	buildCmd.Flags().StringVarP(
+		&memory,
+		"memory",
+		"m",
+		"",
+		"Memory limit for build container. Same as the option for docker.")
```

Comment:
Is it better to add description for format?

Reply:
I will add the unit explain.